dotcom-rendering/src/model/window-guardian.ts

```diff
@@ -17,6 +17,7 @@ export interface WindowGuardianConfig {
 		showRelatedContent: boolean;
 		ajaxUrl: string;
 		shouldHideReaderRevenue: boolean;
+		googleRecaptchaSiteKey?: string;
```

Comment:
Where is this getting set?

Reply:
I think it's this module in frontend that sets gets the values from configuration so they can be added to `window.guardian` (and used elsewhere) but I'm not clear on the mechanism. The actual value of the site key is ultimately generated by google. https://github.com/guardian/frontend/blob/bb3a156ac26b38da78cec3a2230b12d5a2095215/common/app/common/configuration.scala

It's already being put on the `window.guardian` article pages - I may have this wrong, but I assume it just handled been included on the interface yet as no components required it before.